### ISSUE TITLE
android sdk>=35 系统，报RheaTrace : Error: download file binder.txt error 错误，低版本无此异常

### ISSUE THREAD

**Issue by yongyuandeziri** [bytedance/btrace#139]
08-14 16:15:20.563 I RheaTrace : os version is 35. default capture is PerfettoCapture
08-14 16:15:21.482   RheaTrace : start tracing...
08-14 16:15:41.645   RheaTrace : stop tracing...
08-14 16:15:43.852   RheaTrace : MaxAppTraceBufferSize usage 721104/500000000 (0%)
**08-14 16:15:45.930 E RheaTrace : Error: download file binder.txt error**
08-14 16:15:45.930 E RheaTrace :  Tips: check your app is integrated with btrace2.0 and is running.

**Comment by 5peak2me** [comment]
It seems that the Android team changed the `nativeGetEnabledTags` name and parameter of the [method](https://cs.android.com/android/_/android/platform/frameworks/base/+/94f9402fa8f453ce6ea1cbbb32f1f39a574cdec9:core/java/android/os/Trace.java;l=128;bpv=1;bpt=0;drc=500d40b68d14d010a3da1f1f5345b3aa128aaf3d;dlc=8a625b665cfec91d51d6142324d72a00f0d0b7b5)

**Comment by zicen** [comment]
I also encountered the same problem. May I ask how it was finally resolved?

**Comment by WangFeng-AHU** [comment]
> It seems that the Android team changed the name and parameter of the [nativeGetEnabledTags](https://cs.android.com/android/_/android/platform/frameworks/base/+/94f9402fa8f453ce6ea1cbbb32f1f39a574cdec9:core/java/android/os/Trace.java;l=128;bpv=1;bpt=0;drc=500d40b68d14d010a3da1f1f5345b3aa128aaf3d;dlc=8a625b665cfec91d51d6142324d72a00f0d0b7b5) method.

Yes, it is a problem. We will fix it soon, thanks for the feedback!
